spec/controllers/api/v1/webhooks_controller_spec.rb

```diff
@@ -12,6 +12,14 @@
       }.to change(FeedFetcherWorker.jobs, :size).by(4)
       expect_json({ code: 200, errors: [] })
     end
+
+    it 'should be able to enqueue just one FeedFetcher' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.